src/rust/lib/graph-editor/src/component/node/port/output.rs

```diff
@@ -0,0 +1,311 @@
+//! Implements the segmented output port area.
+use crate::prelude::*;
+
+use enso_frp as frp;
+use enso_frp;
+use ensogl::data::color;
+use ensogl::display::Attribute;
+use ensogl::display::Buffer;
+use ensogl::display::Sprite;
+use ensogl::display::scene::Scene;
+use ensogl::display::shape::*;
+use ensogl::display::traits::*;
+use ensogl::display;
+use ensogl::gui::component::Animation;
+use ensogl::gui::component;
+
+use crate::node::NODE_SHAPE_PADDING;
+use crate::node::NODE_SHAPE_RADIUS;
```

Comment:
lets use as few imports as possible. These are good candidates to be used as `node::SHAPE_PADDING` instead :)